versions.tf

```diff
@@ -0,0 +1,4 @@
+
+terraform {
+  required_version = ">= 0.12"
```

Comment:
This highlights that there's currently no version requirement, so I'm going to PR that to master to be explicit that it's currently supporting `~> 0.11`.

Reply:
Awesome. I just verified your PR and rebased my PR. :D

Reply:
Could you move this to be where the current `required_version` specification is in the `main.tf` file.

Reply:
I think I'd also prefer the `~>` syntax as it requires 0.12.x.